task/backend/query_logreader.go

```diff
@@ -109,12 +109,14 @@ func (qlr *QueryLogReader) ListRuns(ctx context.Context, runFilter platform.RunF
 	}
 
 	listScript := fmt.Sprintf(`
+import "influxdata/influxdb/v1"
+
 from(bucketID: "000000000000000a")
   |> range(start: -24h)
 	|> filter(fn: (r) => r._measurement == "records" and r.taskID == %q)
 	|> drop(columns: ["_start", "_stop"])
 	|> group(columns: ["_measurement", "taskID", "scheduledFor", "status", "runID"])
-	|> influxFieldsAsCols()
+	|> v1.fieldsAsCols()
```

Comment:
this is fine.  Maybe we should get one of the task team members to sign off on this?  @mark-rushakoff ?   

I had a related PR where I solved this by replacing with an equivalent pivot()  operation.  whatever the task team wants to use, that's good for me.

Reply:
Fine by me. Tests are still passing.